Fix origin match panic (#412)
silinternational/wecarry-api: README.md

In an appropriate directory:
* git clone git@github.com:silinternational/wecarry-api.git
* cd wecarry-api

> * cp local-example.env local.env

Reviewer: @forevermatt and I were discussing this a few days ago. If we ever want to add a '*.env' to all '.gitignore' files to protect against accidentally committing secrets, this wouldn't work. I think we usually put the `.example` on the end. I'm not totally sold on the idea, since IDEs use the last extension to determine the file type. So use your best judgment until we establish a standard.

* give the entries in local.env the appropriate values

  (note that for deployment to the cloud, you need to provide 

>     (e.g. via an `.env.encrypted` file) the proper settings based

Reviewer: ```suggestion
    in an `.env.encrypted` file the proper settings based
```
to avoid nested parens. You could also use a comma-separate phrase if you prefer that.

>      on the `local-example.env` file)

Reviewer: If you change line 16, remember to change this.
